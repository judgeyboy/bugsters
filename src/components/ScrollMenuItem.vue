<template>
  <li class="nav-item">
    <a v-section-scroll :location="location">
      <slot></slot>
    </a>
  </li>
</template>

<script>
export default {
  name: 'ScrollMenuItem',
  props: ['location']
}
</script>
