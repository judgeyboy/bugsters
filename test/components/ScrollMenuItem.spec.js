import SectionScroll from '@/plugins/section-scroll'
import { shallow, createLocalVue } from 'vue-test-utils'

import ScrollMenuItem from '@/components/ScrollMenuItem'

const localVue = createLocalVue()

localVue.use(SectionScroll)

describe('ScrollMenuItem component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(ScrollMenuItem, {
      localVue
    })
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(ScrollMenuItem, {
      localVue
    })
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
