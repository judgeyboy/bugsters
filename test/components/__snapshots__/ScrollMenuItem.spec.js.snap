// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollMenuItem component should match snapshot 1`] = `"<li class=\\"nav-item\\"><a></a></li>"`;
